### DISCUSSION TITLE
Have you considered Android support?

### DISCUSSION THREAD

**Discussion by TechnicJelle** [amerkoleci/Alimer.Bindings.SDL#10] [Q&A]
Hi there!
I'm about to work on a C# game engine, and I was considering using these SDL3 bindings for it, but I'd like to add Android support to my engine, at some point in the future, so I was wondering if you've thought of adding bindings for Android?
I checked the [runtimes](https://github.com/amerkoleci/Alimer.Bindings.SDL/tree/main/src/Alimer.Bindings.SDL/runtimes) directory, and saw the main desktop platforms already supported, which is already good for the first version of my engine, so it's not a critical feature or anything :)

**Comment by amerkoleci** [comment]
Hi,
Thanks for reaching!
Adding android support shouldn't be that hard but I actually have no plans to support Android (at least not before I tested it).

So if you want to take this implementation I would gladly merge PR in case.

Thanks!
Amer

**Reply by TechnicJelle** [reply]
Thanks for the lighting-fast response!
I'm glad to hear that it's probably not that difficult.
Though I've never properly made bindings for anything myself, before, so it'd probably be a bit harder for me.
Though, if it hasn't been added yet by the time I get to adding Android support to my engine, I'll be sure to look into PR-ing myself!

Would you happen to have a guide about how to add a new platform?
